packages/itwinui-react/src/core/Table/filters/FilterToggle.tsx

```diff
@@ -7,9 +7,9 @@ import cx from 'classnames';
 import type { HeaderGroup } from 'react-table';
 import {
   useGlobals,
-  Popover,
   SvgFilterHollow,
   SvgFilter,
+  Popover,
```

Comment:
nit: maybe undo position change? It was alphabetical before.

Reply:
done